### ISSUE TITLE
Select custom app for opening links

### ISSUE THREAD

**Issue by Lokawn** [Docile-Alligator/Infinity-For-Reddit#817]
Currently external app used for opening links is same as the default browser.

Is it possible to select external app to open links clicked in Infinity instead of the default app?

**Comment by starchyunderscore** [comment]
settibg the app to always open apps of that kind and settibg your browser to open links in app does this

**Comment by Lokawn** [comment]
No, I want to use a different browser than the default browser set in my android for opening links from Infinity.

**Comment by starchyunderscore** [comment]
ah i see, i misinterpreted your comment. i appologize.
